source/rails-girls/_coaches.haml

```diff
@@ -0,0 +1,12 @@
+%section
```

Comment:
Make it a grey section so we have a visual distinction with the section on top of it